01-messaging.md

```diff
@@ -66,6 +67,12 @@ A node:
     - MUST fail the channels.
   - that negotiates an option in this specification:
     - MUST include all the fields annotated with that option.
+  - When defining custom messages:
+   - SHOULD pick a random `type` to avoid collision with other custom types.
+   - SHOULD pick an odd `type` identifiers when regular nodes should ignore the
```

Comment:
The _it's ok to be odd_ rule isn't `type`-specific so there is no need to repeat it here.

Reply:
We chose to repeat it when we spec-ed the onion TLV namespace for experimentation; I did the same here for consistency but can remove it.